src/game/game_state_type.h

```diff
@@ -35,6 +36,23 @@ typedef struct game_player_t game_player;
 typedef struct ticktimer_t ticktimer;
 typedef struct controller_t controller;
 
+// roughly modeled after the configuration in REC files
+typedef struct match_settings_t {
```

Comment:
The _t prefix is technically reserved for stdlib; might be better to just have matching name for struct and typedef.